CONTRIBUTING.md

```diff
@@ -34,6 +34,28 @@ Failure to pass test would result in reverting a change.
 Do not hesitate to ask your colleagues if you need help or have questions.
 Post your question to Telegram or drop a line to cna-storage@vmware.com
 
+## Documentation
+
+Documentation is published to [GitHub
+Pages](https://vmware.github.io/docker-volume-vsphere/) using
+[mkdocs](http://www.mkdocs.org/).
+
+1. Documentation is updated each time a release is tagged.
+2. The latest documentation for the master can be found in
+   [docs](https://github.com/vmware/docker-volume-vsphere/tree/master/doc) in
+   markdown format
+
+To update documentation
```

Comment:
As of now tagging event will not automatically generate the HTML and publish it. Once it is added this step will go away. When we tag we will have to update the GitHub pages branch manually.